TypeScripter.Common/TypeScripter.cs

```diff
@@ -117,11 +117,15 @@ private static int LoadOptions(string[] args)
 						{
 							throw new Exception(String.Format("HttpModule must be one of {0} or {1}", DataServiceGenerator.Http, DataServiceGenerator.HttpClient));
 						}
-						if (!_options.CombineImports)
+						if (!_options.CombineImports.HasValue)
 						{
 							_options.CombineImports = false;
 						}
-					}
+					  if (!_options.HandleErrors.HasValue)
```

Comment:
Since I want to default to true, a default false value will get set to true, so will an explicit false. I used null able bool to get around this.  I changed combine imports as well, for consistency.